src/styles/sass_style/reset.scss

```
@@ -0,0 +1,28 @@
+@use "sass_style/variable.scss";
+
+html {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  text-decoration: none;
+  box-sizing: border-box;
+}
+
+a,
+body,
+h2,
+h3,
+p {
+  margin: 0;
+}
+
+.catalog__container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  margin: 0 auto;
+  width: 1024px;
+}
```

this should not be in the reset file. You can create a separate file for catalog
```suggestion
.catalog__container {
  display: flex;
  flex-wrap: wrap;
  padding: 50px 40px;
  margin: 0 auto;
  width: 1024px;
}
```